src/components/ApplicationsModal/ApplicationsModal.tsx

```diff
@@ -177,6 +183,7 @@ export const ApplicationsModal: React.FC<ApplicationsModalProps> = ({
         appAvatarURL: mostRecentlySelectedAppDetails.logo ? mostRecentlySelectedAppDetails.logo : "",
         appClientID: mostRecentlySelectedAppDetails.clientId ? mostRecentlySelectedAppDetails.clientId : "",
         appClientSecret: mostRecentlySelectedAppDetails.clientSecret ? mostRecentlySelectedAppDetails.clientSecret : "",
+        appDirectURL: mostRecentlySelectedAppDetails.directUrl ? mostRecentlySelectedAppDetails.directUrl : "",
```

Comment:
You can use [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)
```suggestion
        appDirectURL: mostRecentlySelectedAppDetails.directUrl ?? "",
```